autometa/taxonomy/ncbi.py

```diff
@@ -42,7 +42,8 @@
 logger = logging.getLogger(__name__)
 
 BASE_DIR = os.path.dirname(os.path.dirname(__file__))
-NCBI_DIR = os.path.join(BASE_DIR,'databases','ncbi')
+NCBI_DIR = os.path.join(BASE_DIR, 'databases', 'ncbi')
```

Comment:
Perhaps double check here that `NCBI_DIR` is returning `~/Autometa/autometa/databases/ncbi` and not `~/Autometa/databases/ncbi`